mandatory/2-logic-error.js

```diff
@@ -1,18 +1,18 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getWordLength(word) {
-  return "word".length();
+  return word.length;
 }
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  return a * b * c;
 }
 
+const { strict } = require("assert");
```

Comment:
This line loads the module `assert` into the variable `strict`, but it looks like the latter is not used in the code. I would remove the line to make the code cleaner.